hathor/transaction/storage/transaction_storage.py

```diff
@@ -624,6 +624,11 @@ def get_best_block_tips(self, timestamp: Optional[float] = None, *, skip_cache:
             elif meta.score > best_score:
                 best_score = meta.score
                 best_tip_blocks = [block_hash]
+
+        # XXX: if there's more than one we filter it so it's the smallest hash
+        if len(best_tip_blocks) > 1:
```

Comment:
I think that could make sense as a new method, like `get_all_block_tips`, instead of `get_best_block_tips`. But currently that method wouldn't be used anywhere else, and when removing the tips indexes, it would be more convenient if we didn't have to maintain that method.